infinispan-embedded/src/main/java/org/hibernate/ogm/datastore/infinispan/persistencestrategy/counter/ClusteredCounterCommand.java

```diff
@@ -71,6 +75,16 @@ public Number nextValue(NextValueRequest request) {
 
 	}
 
+	private void validateGlobalConfiguration() {
+		boolean globalConfigIsEnabled = cacheManager.getGlobalComponentRegistry()
+			.getGlobalConfiguration().globalState()
```

Comment:
I guess this works but is it possible to have the globalState without setting the persitent-location?

Reply:
we have to try... but it is possible. I'll prepare a test for it.

Reply:
if the persistence location is not present under global state the infinispan.xml file is not parsable by xsd, but counters are stored under project folder.

Reply:
ok